src/common/base/src/base/runtime_tracker.rs

```diff
@@ -245,10 +265,12 @@ impl MemoryTracker {
     }
 }
 
+/// A [`Future`] that enters its thread tracker when being polled.
+///
+/// [`Future`]: std::future::Future
```

Comment:
`rustdoc` is smart enough that We don't need this line if we have imported `Future`.

Reply:
For example: https://opendal.databend.rs/src/opendal/io.rs.html#29

Reply:
Roger that. Next PR.